### DISCUSSION TITLE
Get list of affected paths when creating a PR

### DISCUSSION THREAD

**Discussion by sbanoseg** [community/community#35152] [Actions]
Hello team, I was wondering if anyone of you can help me out to figure this out, currently I get a workflow triggered when a pull request is created, edited, synchronized or reopened and which is formed by 5 jobs that they run in a sequential way, the problem that I want to tackle here is that the last job needs to be triggered only if a specific path has been affected or modified  by the pull request, is there a way to get a list of the affected paths that I can retrieve during a workflow execution, so I can make some validations rely on the results of the list

**Comment by antonblr** [comment]
@sbanoseg - Is this something you're looking for [push/pull_request target paths](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore)? 

```yaml
on:
  push:
    paths:
      - 'my-subdir/**'
      - '**.py'
    paths-ignore:
      - 'docs/**'
```
This will trigger workflow on every push with changes to any`.py` (excluding `docs/` dir) files and any changes in `my-subdir/`

Otherwise you could run `git` command to get the changed files list, something like:
```
git diff "master..." --name-only --diff-filter=ACM
```

**Reply by sbanoseg** [reply]
thanks for your help @antonblr let me try that, and I will get back with you if I have any questions.

**Reply by sbanoseg** [reply]
@antonblr, qq, what does "master..." means in the command you posted?

**Reply by antonblr** [reply]
It shows the differences between `master` and  _current branch_ starting at the last common commit

**Reply by antonblr** [reply]
@sbanoseg - kindly accept the answer, so it does not remain "Unanswered"

**Comment by sbanoseg** [comment]
Awesome, thanks for all your help it worked for me
